Lib/fontmake/font_project.py

```diff
@@ -644,6 +644,18 @@ def run_from_designspace(
             logger.info('Applying instance data from designspace')
             ufos.extend(apply_instance_data(designspace_path,
                                             include_filenames=filenames))
+            logger.info("Expanding features to instance UFOs")
```

Comment:
Maybe we should only expand if there is any include statement, or put this behind an option?

Reply:
I'll turn it into an option.

Reply:
Dunno. Can change to opportunistically expanding stuff unless someone complains?

Reply:
@miguelsousa would autoexpansion impact you? Adobe is the only other big UFO user I know.

Reply:
What is autoexpansion?

Reply:
The goal is work around the problem of relative feature file `include()` paths that get invalidated when fontmake generates instance UFOs for further processing and puts these somewhere else. One solution is  to manually copy stuff over, the other is parsing the feature file of the source marked with `<features copy="1" />`, resolving all includes and writing the fully expanded feature file to the instance UFOs, which would then be independent of external files: autoexpansion.

Reply:
Out of curiosity, how would it impact you and what is the actual workflow you're using?

Reply:
Look at the Source Sans Pro project